src/client/index.ts

```diff
@@ -74,7 +73,7 @@ class Hammerhead {
     processScript: Function;
     get: Function;
     Promise: any;
-    json: any;
+    json: typeof JSON;
```

Comment:
typeof JSON is a 'object' string.

We can remove this export since we are using a native JSON's methods.

Reply:
> typeof JSON is a 'object' string.

it's true for javascript. But in typescript it's type of the `JSON` object - https://github.com/microsoft/TypeScript/blob/79c72751df2aaed2b2ba8caa4170895c343a3d5b/lib/lib.es5.d.ts#L1052
but I agree with the rest.